.github/workflows/docker_scan.yml

```diff
@@ -0,0 +1,43 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Docker Image CVE Scanner
+on:
+  schedule:
```

Comment:
Can we also add the option to trigger this action manually? This may be useful to check if a fix works.

Reply:
I did that on my fork to test the action. The action failed, is it because it detected a CVE?

Reply:
Yes that's correct it's because of detected CVEs. Seems like these are CVEs in the base image

Reply:
These have been fixed in the base image. Whenever we will rebuild this image, these CVEs will get fixed. I ran the pipeline for latest kafka source code which rebuilt the docker image and ran CVE scan on it in my repo https://github.com/VedarthConfluent/test-github-actions/actions/runs/8394467774

Reply:
Sounds good. I will add a manual option as well. Maybe a different workflow altogether for running manually will be better.